src/image_util.rs

```diff
@@ -41,7 +41,11 @@ pub fn load_from_path(path: &Path) -> ImgUtilResult<Vec<RgbaImage>> {
         .filter_map(|res| res.map_or(None, |e| Some(e.path())))
         .collect::<Vec<_>>();
 
-    files.sort();
+    files.sort_by(|a, b| {
+        let a = a.to_string_lossy().into_owned();
+        let b = b.to_string_lossy().into_owned();
+        natord::compare_ignore_case(&a, &b)
```

Comment:
I think it should be case sensitive? I'd rather have it sort too much instead of potentially having an unstable ordering.

Reply:
That makes sense. Fixed.